lib/analytics-stack.ts

```diff
@@ -159,7 +159,7 @@ export class AnalyticsStack extends cdk.NestedStack {
 
         const athenaQueryRtLocation = cloudFrontAccessLogsBucket.s3UrlForObject(athenaResultsPrefix);
         const transformPartFn = new lambda.Function(this, 'transformPartFn', {
-            runtime: lambda.Runtime.NODEJS_12_X,
+            runtime: lambda.Runtime.NODEJS_14_X,
```

Comment:
should we pick latest v1.x to use `NODEJS_16_X` which has longer support lifecycle?

https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-lambda.Runtime.html#static-nodejs_16_x

Reply:
I am trying to bump cdk version in branch `bump-cdk`.